pkg/ipam-controller/controllers/ippool/ippool.go

```diff
@@ -46,13 +46,15 @@ import (
 type IPPoolReconciler struct {
 	PoolsNamespace string
 	NodeEventCh    chan event.GenericEvent
+	MigrationCh    chan struct{}
 	client.Client
 	Scheme   *runtime.Scheme
 	recorder record.EventRecorder
 }
 
 // Reconcile contains logic to sync IPPool objects
 func (r *IPPoolReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	<-r.MigrationCh
```

Comment:
do we want to use 
```
select {
        case <-r.MigrationCh: 
        case <- ctx.Done():
           return ctrl.Result{}, fmt.Errorf("canceled")
        }

```
or something similar to support clean shutdown?

Reply:
Done